tests/etl/test_loaders.py

```
@@ -0,0 +1,51 @@
+import pathlib
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)